source/tutorials/reproducible-scripts.md

```diff
@@ -35,9 +35,11 @@ It determines which program to use for running the script.
 
 We will use the shebang line `#! /usr/bin/env nix-shell`.
 
-`/usr/bin/env` is a program available on most modern Unix-like operating systems.
+[`env`] is a program available on most modern Unix-like operating systems at the file system path `/usr/bin/env`.
 It takes a command name as argument and will run the first executable by that name it finds in the directories listed in the environment variable `$PATH`.
 
+[`env`]: https://www.gnu.org/software/coreutils/manual/html_node/env-invocation.html
```

Comment:
Very small nitpick: Maybe OpenGroup's specification is more appropiate as not all Unix-like systems use GNU?

https://pubs.opengroup.org/onlinepubs/9699919799/utilities/env.html

Reply:
```suggestion
[`env`]: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/env.html
```